psalm.xml.dist

```diff
@@ -60,11 +60,6 @@
         </DeprecatedClass>
         <DeprecatedMethod>
             <errorLevel type="suppress">
-                <!--
-                    This suppression should be removed after 2022
-                    See https://github.com/doctrine/dbal/pull/4317
-                -->
-                <file name="tests/Functional/LegacyAPITest.php"/>
```

Comment:
I believe the `<directory name="tests" />` below comes from a merge conflict improperly resolved in [4532](https://github.com/doctrine/dbal/pull/4532/files#diff-2d20a98059390e83ae58234d4bda07fe9319b822009ce8c4479e0f3ae1943fbfR70).

In newer branches, I'd rather have all deprecations listed here explicitly in order to make sure we remove the BC layer in the next major release. Please revert the removal of the suppressions that don't belong to this patch, remove `<directory name="tests" />` and add the suppressions for this patch. If there are too many new issues reported, you can restore `<directory name="tests" />` after that. We can remove it separately.

Reply:
> 68 errors found

Too many errors, I am restoring `<directory name="tests" />`.